docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,40 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
-- **Refresh**: If the project gets out of sync, use this option to refresh it. To refresh all projects for a connection, right click on the connection and select **Refresh**.
+- **Show all logs**: If log files are available, this action opens all available log files.  
+   - In Eclipse, this action is **Show Log Files>Show All**.
+   - In Eclipse Che and VS Code, this action is **Show all logs**.
+   - In IntelliJ, this action is **Log Files>Show All**.
+- **Hide all logs**: Remove all log files from the **Console** view.
+   - In Eclipse, this action is **Show Log Files>Hide All**.
+   - In Eclipse Che and VS Code, this action is **Hide all logs**.
+   - In IntelliJ, this action is **Log Files>Hide All**.
+- **Manage logs**: In VS Code, this action opens the Command Palette, where you can select which logs you want to see.
+- **Restart in Run Mode**: Restarts the application in run mode. This action is available only on projects that support restart in run mode. 
+- **Restart in Debug Mode**: Restarts the application in debug mode and attaches the debugger. This action is available only on projects that support it. For MicroProfile and Spring projects, a debug session is started automatically. For Node.js projects, you are guided in the setup of a debug session in a Chromium-based browser. For all other project types that support debug, set up a debug session manually. 
+- **Attach Debugger**:  If you accidentally detached the Java debugger or restarted your IDE, use this action to set up a new debug session. This action is available only for Java applications that support restart in debug mode.
+- **Launch Debug Session**: If you ended your Node.js debug session accidentally, use this action to set up a new debug session. This menu item is available only for Node.js applications that support restart in debug mode.
+- **Build**: Initiate a build of your project. This action doesn't function if a build is already running. If you try to use it when a build is running, it doesn't work, or a warning message appears.
+- **Disable Auto Build** and **Enable Auto Build**: Enable or disable the automatic building of projects. Use this action to disable automatic builds if you are making a lot of changes and don't want builds to be triggered until you are done. This action is enabled by default.
+- **Enable Inject Appmetrics** and **Disable Inject Appmetrics**: Enable or disable the injection of application metrics at build time. **Enable Inject Appmetrics** provides monitoring capabilities that the Performance Dashboard uses. It is disabled by default and is supported on Codewind-style Java, Node.js, and Swift projects, except Java Lagom.
+- **Move Project**: Move a project to a different workspace.
```

Comment:
@sujeilyfonseca, I wasn't sure about the "Move Project" action. Please let me know if this line needs changing. Thank you!

Reply:
"Move Project" is currently available in VS Code, but we shouldn't be IDE-specific, as this might later be in other IDEs. It moves a project to a different Codewind connection.

Reply:
I think it's OK to mention that the action is currently available in VS Code so users won't get puzzled if they don't see it in other IDEs. If we eventually add it to all IDEs, we can just remove the "In VS Code" text whenever it becomes available in all IDEs. :)

Reply:
That's a good point! :)